lightning/src/ln/functional_tests.rs

```diff
@@ -5001,8 +5015,12 @@ fn test_onion_failure() {
 		let cur_height = nodes[0].node.latest_block_height.load(Ordering::Acquire) as u32 + 1;
 		let onion_keys = onion_utils::construct_onion_keys(&Secp256k1::new(), &route, &session_priv).unwrap();
 		let (mut onion_payloads, _htlc_msat, _htlc_cltv) = onion_utils::build_onion_payloads(&route, cur_height).unwrap();
-		onion_payloads[0].format = msgs::OnionHopDataFormat::BogusRealm(3);
-		msg.onion_routing_packet = onion_utils::construct_onion_packet(onion_payloads, onion_keys, [0; 32], &payment_hash);
+		let mut new_payloads = Vec::new();
+		for payload in onion_payloads.drain(..) {
+			new_payloads.push(BogusOnionHopData::new(payload));
+		}
+		new_payloads[0].data[0] = 1;
```

Comment:
The significance of this is unclear to me. Is this the version byte? Can we use a constant for it?

Reply:
Its just to invalidate it, added a comment to note what it does.